include/libgit4cpp/GitRepository.h

```diff
@@ -110,32 +111,32 @@ class GitRepository
     void reset(int nr_of_commits);
 
     /**
-     * push commits to a sequernce repository.
-     * \param addr: address of the git repository host
+     * Push commits to the repository.
+     * \param addr Address of the remote git repository host
     */
     void push();
 
     /**
-     * pull changes from a sequence repository.
-     * \param addr: address of the git repository host
+     * Pull changes from the remote repository.
+     * \param addr Address of the git repository host
     */
     void pull();
 
     /**
-     * clone a sequence repository.
-     * \param addr: address of the git repository host
+     * Clone a repository.
+     * \param addr Address of the git repository host
     */
-    void clone_repo(const std::string& url, const std::filesystem::path& repo_path );
+    void clone_repo(const std::string& url, const std::filesystem::path& repo_path);
 
     /**
      * Check if remote and local branch are in same state.
-     * \param branch_name 
+     * \param branch_name
     */
     bool branch_up_to_date(const std::string& branch_name);
 
     /**
-     * Deletes seq_repository and all files within from git index.
-     * \param directory: directory holding sequence
+     * Deletes  the repository and all files within from the git index.
```

Comment:
Me nitpicking: Should be "Delete the repository..." instead of "Deletes the repository..."

Reply:
And it's even wrong. Thanks for the good find!!